src/base/Assert.hh

```diff
@@ -171,6 +185,42 @@
  */
 #define CELER_CUDA_CHECK_ERROR() CELER_CUDA_CALL(cudaPeekAtLastError())
 
+/*!
+ * \def CELER_MPI_CALL
+ *
+ * When MPI support is enabled, execute the wrapped statement and throw a
+ * RuntimeError if it fails. If MPI is disabled, throw an unconfigured
+ * assertion.
+ *
+ * If it fails, we call \c cudaGetLastError to clear the error code.
+ *
+ * \code
+ *     CELER_CUDA_CALL(cudaMalloc(&ptr_gpu, 100 * sizeof(float)));
+ *     CELER_CUDA_CALL(cudaDeviceSynchronize());
+ * \endcode
```

Comment:
Left in by accident?

Reply:
Yep, copy-paste error on my part. Clearly I don't read code examples...